runtime-modules/content/src/lib.rs

```diff
@@ -1477,7 +1470,7 @@ decl_event!(
         ContentActor = ContentActor<
             <T as ContentActorAuthenticator>::CuratorGroupId,
             <T as ContentActorAuthenticator>::CuratorId,
-            <T as membership::Trait>::MemberId,
+            <T as common::MembershipTypes>::MemberId,
```

Comment:
Maybe I'm not getting it. Shouldn't we use exclusively MemberId out of the membership pallet.
Since it is also used in the storage module+